src/constants/network.js

```diff
@@ -1,7 +1,12 @@
 import { Networks } from "stellar-sdk";
+import { Networks as SorobanNetworks } from "soroban-sdk";
 
 const NETWORK = {
   available: {
+    futurenet: {
+      horizonURL: "https://horizon-futurenet.stellar.org",
+      networkPassphrase: SorobanNetworks.FUTURENET,
+    },
```

Comment:
this is the real functional change in this PR: adding a `futurenet` button to the top of Lab for ease of testing Soroban